pyout/field.py

```diff
@@ -487,7 +487,7 @@ class PlainProcessors(StyleProcessors):
     """Ignore color, bold, or underline styling.
     """
 
-    style_types = {}
+    style_types = OrderedDict()
```

Comment:
oh, old times... I guess could even RF to get away from using it in favor of regular `dict` which is ordered now AFAIK, although might better be a separate PR to ensure there is no side effects

Reply:
Looked into this very briefly-- I guess we probably do want to stay with OrderedDicts, if only for the re-ordering functionality.

Reply:
not sure what reordering functionality is used, even if any, I guess could be replicated with a regular `dict`.